src/btouch.cs

```diff
@@ -122,20 +122,25 @@ static int Main (string [] args)
 		}
 	}
 
-#if !MONOMAC
 	static string GetSDKRoot ()
 	{
-		var sdkRoot = Environment.GetEnvironmentVariable ("MD_MTOUCH_SDK_ROOT");
-		if (string.IsNullOrEmpty (sdkRoot)) {
-			if (Directory.Exists ("/Library/Frameworks/Xamarin.iOS.framework/Versions/Current")) {
-				sdkRoot = "/Library/Frameworks/Xamarin.iOS.framework/Versions/Current";
-			} else {
-				sdkRoot = "/Developer/MonoTouch/usr";
+		switch (CurrentPlatform) {
+		case PlatformName.iOS:
+		case PlatformName.WatchOS:
+		case PlatformName.TvOS:
+			var sdkRoot = Environment.GetEnvironmentVariable ("MD_MTOUCH_SDK_ROOT");
+			if (string.IsNullOrEmpty (sdkRoot)) {
+				if (Directory.Exists ("/Library/Frameworks/Xamarin.iOS.framework/Versions/Current")) {
+					sdkRoot = "/Library/Frameworks/Xamarin.iOS.framework/Versions/Current";
+				} else {
+					sdkRoot = "/Developer/MonoTouch/usr";
```

Comment:
Not rally related to the `#if` removal but does it make sense to keep this (6.x and earlier) logic around ?

It looks like we should be checking that `MD_MTOUCH_SDK_ROOT` directory exists (if not null/empty) and fallback to well known path